src/wheatley.ts

```diff
@@ -80,9 +82,16 @@ type text_command_map_target = {
     deletable: boolean;
 };
 
+type authentication = {
+    id: string;
```

Comment:
Would it be possible to just get the client's id after ready rather than specifying it in the json